app/controllers/tpi/companies_controller.rb

```diff
@@ -3,7 +3,20 @@ class CompaniesController < ApplicationController
     def show
       @company = Company.find(params[:id])
 
-      @company_details = ::Api::Presenters::Company.new(@company).company_details
+      @company_summary = company_presenter.summary
+      @company_mq_assessments = company_presenter.mq_assessments
+    end
+
+    # Data:     Company MQ Assessments Levels over the years
+    # Section:  MQ
+    # Type:     line chart
+    # On pages: :show
+    def assessments_levels_chart_data
+      @company = Company.find(params[:id])
```

Comment:
Maybe we could move that into `before_action :fetch_company`

Reply:
:railway_track: fixed